099_1_bfs.cpp

```diff
@@ -0,0 +1,35 @@
+int n;                 // viršūnių skaičius
+vector<int> adj[MAXN]; // kaimynystęs sąrašas, kur adj[i] yra i-tosios viršūnės kaimynų sąrašas

```

Comment:
```suggestion
vector<int> adj[MAXN]; // kaimynystės sąrašas, kur adj[i] yra i-tosios viršūnės kaimynų sąrašas

```

Bet kaip minėjau anksčiau, šitų eilučių manyčiau, kad nereikia.